src/platform/i386/boot_comp.c

```diff
@@ -155,8 +158,12 @@ kern_boot_comp(void)
 	 * Need to account for the pages that will be allocated as
 	 * PTEs
 	 */
+	local_pgd = (pgtbl_t)mem_boot_alloc(1);
+	local_pgd = (pgtbl_t)chal_va2pa(local_pgd);
+        if (pgtbl_activate(ct, BOOT_CAPTBL_SELF_CT, BOOT_CAPTBL_SELF_LOCAL_PT, local_pgd, 0)) assert(0);
```

Comment:
I should be more clear:  There are spaces here instead of tabs.  There likely should not be.